pkg/web/src/app/common/components/ConfirmModal.tsx

```diff
@@ -7,6 +7,7 @@ import { UnknownMutationResult } from '../types';
 
 export interface IConfirmModalProps {
   variant?: ModalProps['variant'];
+  confirmationVariant?: 'danger' | 'default';
```

Comment:
`confirmationVariant` makes me fill uneasy, a. because `titleIconVariant` can actually be `danger`, b. when we use it, we do `confirmationVariant === 'danger'` that means we actually only interested in `isDanger`, c. we translate `danger` to `warning` for the title icon without explaining how `...variant='danger'` become `...varian='wanring'`

can we do this instead: ?
```suggestion
  isDanger?: boolean | undefined;
```

Reply:
hmm, if we are not using the `variant` maybe we can ovverride it to be `variant: 'delete' | 'info' | ...` instead of `small/bing` (depending if we use it soemeware for small / big stuff)?

Reply:
The readability issues that you pointed are caused by overriding/re-using the standard enum values (`danger`, `default`...).  Take a look at the newest version - now the `confirmationVariant` has distinct values from `titleIconVariant` and  button `variant`.

Reply:
I still have a problem with the `x === 'danger' ? 'danger' : 'other value'` pattern, can you look for a way around that ?